### ISSUE TITLE
Homebrew based build process fails due to htslib headers

### ISSUE THREAD

**Issue by ifiddes** [vgteam/vg#2673]
```
/Users/ian.fiddes/vg/deps/libvgio/include/vg/io/blocked_gzip_input_stream.hpp:6:25: fatal error: htslib/bgzf.h: No such file or directory
 #include <htslib/bgzf.h>
                         ^
compilation terminated.
make[3]: *** [CMakeFiles/vgio_static.dir/src/basic_stream.cpp.o] Error 1
make[2]: *** [CMakeFiles/vgio_static.dir/all] Error 2
make[1]: *** [all] Error 2
make: *** [lib/libvgio.a] Error 2
```

However, I have the required header in `./include/htslib/bgzf.h`, this just doesn't seem to be part of the build path.

**1. What were you trying to do?**

**2. What did you want to happen?**

**3. What actually happened?**

**4. If you got a line like `Stack trace path: /somewhere/on/your/computer/stacktrace.txt`, please copy-paste the contents of that file here:**

```
Place stacktrace here.
```

**5. What data and command can the vg dev team use to make the problem happen?**

**6. What does running `vg version` say?**

```
Place vg version output here
```

**Comment by adamnovak** [comment]
I don't think this has much to do with Homebrew specifically; we ship our own hstlib and it's the libvgio dependency build that is failing to find it. Plus Travis tests our builds with Homebrew on Mac and it has been working fine.

What does the log say right before this? It should have a compiler command where we might or might not have a `-I` option telling libvgio how to find htslib.

Also, do things work any better if you `. ./source_me.sh` in the main vg directory, before the build?

**Comment by faithokamoto** [comment]
If you still have a problem with updated vg, please open a new Issue with such.
